packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -60,8 +59,10 @@
     w-full
     overflow-visible
     align-top;
+}
 
-  box-shadow: var(--calcite-input-date-picker-shadow, var(--calcite-shadow-none));
+:host .menu-container .calcite-floating-ui-anim {
+  box-shadow: var(--calcite-input-date-picker-calendar-shadow, var(--calcite-shadow-none));
```

Comment:
Should this be `--calcite-shadow-sm` ?

Reply:
Yes! 2 separate tokens, input set to none, calendar set to small. I made the change. Thanks for catching that!